policies/Sponsorship_Policy.md

```diff
@@ -0,0 +1,153 @@
+# Sponsorship Policy
+
+## Purpose
+
+The purpose of the Sponsorship Policy (The Policy) is to outline the principles
+ and behaviours adopted by OpenSSL when receiving or providing sponsorship.  
+
+The Policy will ensure that only appropriate sponsorship is received or 
+provided by OpenSSL, i.e. that sponsorship aligns with OpenSSL's Mission and values, 
+community expectations and meets legal requirements.
+
+
+## Scope
+
+The policy applies to all Organisations, individuals and entities that provide 
+sponsorship to OpenSSL and to any organisation, individual or entity that 
+OpenSSL sponsors.
+
+The Policy establishes a framework by which new and renewing sponsorships are 
+assessed, formalised, managed, and measured for both the sponsorship of OpenSSL
+ and any sponsorship that OpenSSL may provide.  
+
+
+## Principles
+
+### Definition
+
+**Sponsorship** is the provision of support to an organisation, individual or 
+entity from an external source, it can take many forms including but not 
+limited to, financial assistance, preferential treatment, free products, 
+resources or beneficial "in kind" arrangements.
+
+
+### Levels of Sponsorship provided to OpenSSL
+Details of Sponsorship options are located on OpenSSL's [Sponsorship webpage].
+
+
+### Sponsorship Decisions
+
+Only the OMC, in line with The Policy, can decide whether to accept sponsorship
+ or to provide sponsorship.
+Sponsorship provided via [GitHub Sponsors] in part or in whole, is deemed to be
+ accepted sponsorship by the OMC, however the OMC reserves the right to reject 
+any sponsorship provided via the [GitHub Sponsors] where it is deemed by the 
+OMC, to be inappropriate.
```

Comment:
are we able to deny specific github sponsors? I don't believe that's possible (needs check)